src/main/java/frc/robot/systems/EveryBotIntakeFSM.java

```diff
@@ -252,30 +273,41 @@ private EveryBotIntakeFSMState nextState(TeleopInput input) {
 						avgcone += currLogs[i];
 					}
 					avgcone /= AVERAGE_SIZE;
-					if (avgcone > CURRENT_THRESHOLD || !(input.isIntakeButtonPressed())) {
+					if (avgcone > CURRENT_THRESHOLD) {
+						holding = true;
 						return EveryBotIntakeFSMState.IDLE_STOP;
+					} else {
+						holding = false;
 					}
 				}
+				forward = input.isThrottleForward();
```

Comment:
I'm feeling that `isThrottleForward` isn't the clearest name for what this actually does. The code in TeleopInput is responsible for taking care of what button is mapped to what action. The API it exposes should describe what info the subsystem wants to know from TeleopInput, not what button happens to be mapped to those actions.